football/lib/flagMapper.js

```diff
@@ -0,0 +1,72 @@
+const flagForTeam = (teamName) => {
+    switch (teamName) {
+        case `Russia`:
+            return `🇷🇺`
+        case `Saudi Arabia`:
+            return `🇸🇦`
+        case `Egypt`:
+            return `🇪🇬`
+        case `Uruguay`:
+            return `🇺🇾`
+        case `Morocco`:
+            return `🇲🇦`
+        case `Iran`:
+            return `🇮🇷`
+        case `Portugal`:
+            return `🇵🇹`
+        case `Spain`:
+            return `🇪🇸`
+        case `France`:
+            return `🇫🇷`
+        case `Australia`:
+            return `🇦🇺`
+        case `Argentina`:
+            return `🇦🇷`
+        case `Iceland`:
+            return `🇮🇸`
+        case `Peru`:
+            return `🇵🇪`
+        case `Denmark`:
+            return `🇩🇰`
+        case `Croatia`:
+            return `🇭🇷`
+        case `Nigeria`:
+            return `🇳🇬`
+        case `Costa Rica`:
+            return `🇨🇷`
+        case `Serbia`:
+            return `🇷🇸`
+        case `Germany`:
+            return `🇩🇪`
+        case `Mexico`:
+            return `🇲🇽`
+        case `Brazil`:
+            return `🇧🇷`
+        case `Switzerland`:
+            return `🇨🇭`
+        case `Korea Republic`:
+            return `🇰🇷`
+        case `Belgium`:
+            return `🇧🇪`
+        case `Panama`:
+            return `🇵🇦`
+        case `Tunisia`:
+            return `🇹🇳`
+        case `Sweden`:
+            return `🇸🇪`
+        case `Poland`:
+            return `🇵🇱`
+        case `England`:
+            return `🇬🇧`
```

Comment:
I think for England it should be this flag: https://emojipedia.org/flag-for-england/ :)

Reply:
raised an issue on the main repo https://github.com/novoda/dashboards/issues/38